lib/trento_web/plugs/api_redirector.ex

```diff
@@ -23,34 +27,43 @@ defmodule TrentoWeb.Plugs.ApiRedirector do
 
   @impl true
   def call(%Plug.Conn{path_info: [_ | path_parts], method: method} = conn, opts) do
-    latest_version = Keyword.get(opts, :latest_version)
+    available_api_versions = Keyword.get(opts, :available_api_versions)
     router = Keyword.get(opts, :router)
 
-    unless latest_version do
-      raise ArgumentError, "expected :latest_version option"
+    unless available_api_versions do
```

Comment:
Maybe we can add the same sanity check even if the list is present but empty, to avoid strange behaviour like getting all 404

Wdyt?

Reply:
@CDimonaco sanity check added.
Besides that, I have moved the argument validation to the `init` function which looks the more appropriate place